code/modules/mob/living/carbon/human/species.dm

```diff
@@ -164,11 +164,16 @@
 
 	if(exotic_bloodtype && C.dna.blood_type != exotic_bloodtype)
 		C.dna.blood_type = exotic_bloodtype
-
+	if(C.dna.features["legs"] == "Digitigrade Legs")
+		specflags += DIGITIGRADE
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(0)
 
 /datum/species/proc/on_species_loss(mob/living/carbon/C)
 	if(C.dna.species.exotic_bloodtype)
 		C.dna.blood_type = random_blood_type()
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(1)
```

Comment:
Yes, it would give identical context as ```1```/```0``` does, except we don't allow 1s/0s to be used for bools when we have the ```#define```'d versions.

it's just a style thing, but if you don't do it I'll call oranges and he'll really get anal about true false.